<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skilld invoice settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtBillingName"/>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>792</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>350</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Remove</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridOpts">
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="btnAddBankItem">
        <property name="text">
         <string>Add item</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="btnSaveInvoiceSettings">
      <property name="minimumSize">
       <size>
        <width>228</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Save settings</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelBillingName">
      <property name="text">
       <string>Billing name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelBankInfo">
      <property name="text">
       <string>Bank information:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
